.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,81 @@
+---
+
+name: Pre-Release Issue Template
+about: Should be used when Preparing for a release to make sure no tasks are missed during the build-up for a release
+title: 'Mantid Imaging [V.V.V] Pre-Release'
+labels: 'Release 🚀'
+assignees: ''
+
```

Comment:
If this shows up as a template when creating a new issue, is there a way to have it only visible to devs?

Reply:
> I would be in favour of just leaving public
That's fine

Reply:
I've updated the title to hopefully make the template obvious that it is for developers only